lib/ansible/plugins/connection/__init__.py

```diff
@@ -120,6 +123,21 @@ def _split_ssh_args(argstring):
             # In Python3, shlex.split doesn't work on a byte string.
             return [to_text(x.strip()) for x in shlex.split(argstring) if x.strip()]
 
+    @staticmethod
```

Comment:
I think this makes more sense in a shared utility module, rather than in the base connection (or even just hang it off the powershell shell plugin for now).